examples/tv-casting-app/linux/CastingUtils.cpp

```diff
@@ -0,0 +1,120 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "CastingUtils.h"
+
+CHIP_ERROR DiscoverCommissioners()
```

Comment:
Any reason not to put CastingUtils into tv-casting-common?

Reply:
I guess these are just wrappers for functionality that can be invoked from android/iOS bypassing this file

Reply:
Yes. CastingUtils is supposed to be a set of linux specific helper utils for main.cpp/CastingShellCommands.cpp to call. It has functions like InitCommissioningFlow that print out the commissioners on the Linux CLI, which will not be applicable for Android/iOS. So I kept it under linux instead of tv-casting-common. They are wrappers around more basic commands that are indeed exposed over CastingServer.h/.cpp. The wrappers themselves have a Linux CLI element to them or are calling other functions that have logic specific to the Linux CLI.